### PR TITLE
functional profile designer

### PR DESCRIPTION
**Pull request by devksingh4**

### CODE REVIEW

File: clientv2/src/components/ProfileViewer/options.ts [hunk 1]
```diff
@@ -0,0 +1,311 @@
+export const institutionOptions = [
```
Comment (jlevine18) [inline clientv2/src/components/ProfileViewer/options.ts:1]: where did you get this list from?

File: clientv2/src/components/ProfileViewer/options.ts [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  'Computer Science + Economics',
+  'Computer Science + Geography & Geographic Information Science',
+  'Computer Science + Linguistics',
+  'Computer Science + Philosophy',
+  'Computer Science + Advertising',
+  'Mathematics & Computer Science',
+  'Statistics & Computer Science',
+];
+
+export const degreeOptions = [
```
Comment (jlevine18) [inline clientv2/src/components/ProfileViewer/options.ts:73]: Do we just want to break this up into "Bachelors", "Non-thesis Masters", "Researcb Masters", and "PhD"? I don't think people care about BSLAS vs BS vs BFA, and tbh i would concurrently list BS and MCS if I were BS/MCS

Comment (jlevine18) [reply clientv2/src/components/ProfileViewer/options.ts:73]: Also do non-FAANG recruiters even know what MS vs MCS is? (that's why "research" vs "non-research" master's may be the easier term)

**Review by jlevine18 — APPROVED** [review]
as usual, this looks great. might be a bit more complicated than we need, especially with categorizing majors. we miss a few edge cases this way (dual-degree/dual-major/joint grad programs/transfers from il ccs), and idt people really know or care that cs+music is technically a bfa program than a bs program. i'd consider changing to just text fields, unless you think there's a compelling reason to stick with this system?

File: clientv2/src/components/ProfileViewer/options.ts (hunk 1 above)
Comment (devksingh4) [reply clientv2/src/components/ProfileViewer/options.ts:1]: It's just a list of common institutions for the auto complete. It's not the only options.

File: clientv2/src/components/ProfileViewer/options.ts (hunk 2 above)
Comment (devksingh4) [reply clientv2/src/components/ProfileViewer/options.ts:73]: We can split into bachelors, masters thesis, masters non-thesis, PhD.

Comment (jlevine18) [reply clientv2/src/components/ProfileViewer/options.ts:73]: I think that's the best split for search purposes. Other than places that need ABET accreditation, that's the 4 categories that recruiters will search by.

File: clientv2/src/components/ProfileViewer/options.ts (hunk 1 above)
Comment (jlevine18) [reply clientv2/src/components/ProfileViewer/options.ts:1]: ok, perfect

**Pull request merged**